pkg/analyser/wechat/wechat.go

```diff
@@ -93,15 +93,21 @@ func (w Wechat) GetAccountsAndTags(o *ir.Order, cfg *config.Config, target, prov
 		if r.Time != nil {
 			match, err = util.SplitFindTimeInterval(*r.Time, o.PayTime, match)
 			if err != nil {
-				log.Fatalf(err.Error())
+				log.Fatalf("Error in SplitFindTimeInterval: %v", err)
 			}
 		}
 		if r.TimestampRange != nil {
 			match, err = util.SplitFindTimeStampInterval(*r.TimestampRange, o.PayTime, match)
 			if err != nil {
-				log.Fatalf(err.Error())
+				log.Fatalf("Error in SplitFindTimeStampInterval: %v", err)
 			}
 		}
+		if r.MinPrice != nil && o.Money < *r.MinPrice {
```

Comment:
what is the expected behavior if two min prices overlap? Let's say we mistakenly have the following rules :
```yaml
    - category: 日用百货
      minPrice: 10
      targetAccount: Expenses:Groceries
    - category: 日用百货
      minPrice: 11
      maxPrice: 99.99
      targetAccount: Expenses:Food:Drink
```

if the grocery item in the bill costs 12, it matches both rules, but we will the pick this rule `targetAccount: Expenses:Food:Drink`, not sure whether it's the expected behavior.

Reply:
This is the expected behavior, we have already described this behavior on README.

<img width="902" alt="image" src="https://github.com/user-attachments/assets/553e55bb-319a-47d9-81d8-6fc9152b723b">